src/lib.rs

```diff
@@ -57,6 +57,45 @@ pub struct ActiveTransformFeedback {
     pub name: String,
 }
 
+#[derive(Debug)]
+pub struct ShaderPrecisionFormat {
+    /// The base 2 log of the absolute value of the minimum value that can be represented
+    pub range_min: i32,
+    /// The base 2 log of the absolute value of the maximum value that can be represented.
+    pub range_max: i32,
+    /// The number of bits of precision that can be represented.
+    /// For integer formats this value is always 0.
+    pub precision: i32,
+}
+
+impl ShaderPrecisionFormat {
+    /// Returns OpenGL standard precision that most desktop hardware support
+    pub fn default(precision_type: u32, is_embedded: bool) -> Self {
```

Comment:
If we're saying this might be a good guess at a desktop baseline, then we might want to give it a specific name in case WebGL projects rely on this, like `typical_desktop_hardware` or something. What do you think?

Reply:
Makes sense.